src/native_driver.jl

```diff
@@ -179,4 +205,69 @@ function Distributed.manage(manager::K8sClusterManager, id::Integer, config::Wor
             notify(DEREGISTER_ALERT; all=true)
         end
     end
+
+    return nothing
+end
+
+# Stripped down and modified version of:
+# https://github.com/JuliaLang/julia/blob/844c20dd63870aa5b369b85038f0523d7d79308a/stdlib/Distributed/src/managers.jl#L567-L632
+function Distributed.connect(manager::K8sClusterManager, pid::Int, config::WorkerConfig)
```

Comment:
just to check my own understanding, we need to provide a specialized method for this because of the need to setup port forwarding in the case of a local-to-cluster connection?

Reply:
Correct. This is [actually supported](https://github.com/JuliaLang/julia/blob/f7554b5c9f0f580a9fcf5c7b8b9a83b678e2f48a/stdlib/Distributed/src/managers.jl#L556-L566) but it's not well documented.